pubspec.yaml

```diff
@@ -58,8 +58,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+    - assets/
```

Comment:
prefer .webp over .png
.webp file takes less size than .png 

This is done to minimise the app size by reducing the space occupied by assets